### ISSUE TITLE
Fix Navbar

### ISSUE THREAD

**Issue by Diegopie** [Diegopie/sapa-nextjs#1]
[] The mobile menu is ugly and doesn't have proper keyboard nav,
[] Logo does not center bottom
[] May also add skip link
